web/js/userTasksReport.js

```diff
@@ -161,7 +197,33 @@ Ext.onReady(function () {
         width: 350,
         renderTo: 'content',
         defaults: {width: 230},
-        items: [{
+        items: [
+            new Ext.form.ComboBox({
+                fieldLabel: 'User',
+                autoLoad: true,
+                typeAhead: true,
+                mode: 'local',
+                store: usersStore,
+                valueField: 'id',
+                displayField: 'login',
+                triggerAction: 'all',
+                forceSelection: true,
+                id: 'userLogin',
+                baseParams: {
+                    'userId' : userId
+                },
+                listeners: {
+                    'select': function () {
+                        this.baseParams.userId = this.getValue();
+                    },
+                    'blur': function () {
+                        if(this.baseParams.userId != this.getValue()){
+                            this.baseParams.userId = this.getValue();
+                        }
+                    }
```

Comment:
You don't need these listeners here. Look at the implementation of the customers combo, for example; the value of the combo is retrieved later in `findTasks()`.